explore.py

```diff
@@ -77,16 +79,43 @@ def main(args):
     print("Percent of total revenue attributed to sessions using a particular device: \n",
           explore_utils.find_percent_of_total_revenue_by_device(data))
 
-    # Prints out the percent of sessions accessed via a particular device taht
+    # Prints out the percent of sessions accessed via a particular device that
     # generated revenue
     print("Percent of total sessions using a particular device that generated revenue: \n",
           explore_utils.find_percent_device_uses_generating_revenue(data))
 
+    # Plots (1) monthly and (2) yearly histograms of number of site visits,
```

Comment:
Please add a command-line argument (or multiple) that allows the user to select whether or not to display these graphs, and also whether or not to save them to disk.